### PR TITLE
Predictions

### PR DESCRIPTION
**Pull request by hbalenda**
```
Hannahs-Macbook-Pro-2:flat_circle hbalenda$ curl -H "Authorization: Token http://localhost:3000/api/predictions/3
["iPhone","bitters"]
```

The curl request for predictions works!!! 
Coverage 93.09%

### CODE REVIEW

**Comment by hbalenda** [comment]
I'm not sure what's going on with Travis. It's erroring out at rake app:db:create.

**Comment by brittanyjspringer** [comment]
@hcbviolet Please remove the 'app' from the Spec line too. That is my fault -- I gave you the Travis config to test a Rails Engine vs. a full Rails app. Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - 2.2.4
+
+services: postgres
+
+before_script:
+  - bundle exec rake db:create
+  - bundle exec rake db:migrate
+
+script:
+  - bundle exec rake app:spec
```
Comment (brittanyjspringer) [inline .travis.yml]: Remove app from this line, please.

File: app/controllers/api/predictions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Api::PredictionsController < ApiController
+  before_action :authenticate_user
+
+  def show
+    @prediction = Prediction.find(params[:id])
+    @year = @prediction.year
+    arr = []
+    Trend.all.each do |trend|
```
Comment (brittanyjspringer) [inline app/controllers/api/predictions_controller.rb:8]: So clean! 👍

**Comment by hbalenda** [comment]
Ok @Brit200313! After 8 million commits, I got rubocop and travis configured. What're your thoughts on the two remaining rubocop offenses (method has too many lines of code and Assignment Branch Condition size too high)?

**Comment by brittanyjspringer** [comment]
I'm fine with them. Rubocop is an opinion, not the law. Merging!

**Pull request merged**